Microsoft.Toolkit.Uwp.Services/Services/Twitter/TwitterDataProvider.cs

```diff
@@ -35,7 +36,7 @@ namespace Microsoft.Toolkit.Uwp.Services.Twitter
     /// <summary>
     /// Data Provider for connecting to Twitter service.
     /// </summary>
-    public class TwitterDataProvider : Toolkit.Services.DataProviderBase<TwitterDataConfig, Tweet>
+    public class TwitterDataProvider : Toolkit.Services.DataProviderBase<TwitterDataConfig, Toolkit.Services.SchemaBase>
     {
```

Comment:
private static HttpClient client; 
should be 
private static HttpClient _client;
I know it something old but it will cause some warnings